### ISSUE TITLE
docs: 📚 sd-audio docs update + template

### ISSUE THREAD

**Issue by MartaPintoTeixeira** [solid-design-system/solid#1581]
## Description
Update docs according to Figma:
- transcript icon should be visible in all samples (as it is mandatory for accessibility reasons)
- create template for sd-audio (see figma)

## Where to put the documentation
- storybook sd-audio

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by martascorreia** [comment]
@MartaPintoTeixeira I think both points are done, could you confirm?

**Comment by yoezlem** [comment]
Ping @MartaPintoTeixeira
